### PR TITLE
currency_rate_date_check: port to v8

### PR DESCRIPTION
**Pull request by alexis-via**

### CODE REVIEW

File: currency_rate_date_check/__openerp__.py [hunk 1]
```diff
@@ -31,10 +31,9 @@
 Currency Rate Date Check
```
Comment (pedrobaeza) [inline currency_rate_date_check/__openerp__.py:31]: Please put this in a README.rst file.

File: currency_rate_date_check/currency_rate_date_check.py [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+                    # now we do the real work !
+                    cr.execute(
+                        'SELECT rate, name FROM res_currency_rate '
+                        'WHERE currency_id = %s '
+                        'AND name <= %s '
+                        'ORDER BY name desc LIMIT 1',
+                        (currency_id, date))
+                    if cr.rowcount:
+                        rate_date = cr.fetchone()[1]
+                        rate_date_dt = datetime.strptime(
```
Comment (pedrobaeza) [inline currency_rate_date_check/currency_rate_date_check.py]: You can use `fields.Datetime.from_string()`

File: currency_rate_date_check/currency_rate_date_check.py [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+                            rate_date, DEFAULT_SERVER_DATETIME_FORMAT)
+                        if len(date) <= 10:
+                            date_dt = datetime.strptime(
+                                date, DEFAULT_SERVER_DATE_FORMAT)
```
Comment (pedrobaeza) [inline currency_rate_date_check/currency_rate_date_check.py]: You can use `fields.Date.from_string()`

File: currency_rate_date_check/currency_rate_date_check.py [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+                        else:
+                            date_dt = datetime.strptime(
+                                date, DEFAULT_SERVER_DATETIME_FORMAT)
```
Comment (pedrobaeza) [inline currency_rate_date_check/currency_rate_date_check.py]: You can use `fields.Datetime.from_string()`

**Comment by pedrobaeza** [comment]
Change icon path from /static/src/img/icon.png to /static/description/icon.png.

**Comment by pedrobaeza** [comment]
Travis error seems unrelated

**Comment by tafaRU** [comment]
@pedrobaeza could you please explain to me the reason of https://github.com/OCA/account-financial-tools/pull/118#issuecomment-68516492 ?

Thank you in advance.

**Comment by pedrobaeza** [comment]
In v8, the new location where Odoo looks for the icon is `/static/description/icon.png`

**Comment by tafaRU** [comment]
@pedrobaeza  thanks a lot for your clarification :smiley:

**Comment by pedrobaeza** [comment]
Travis errors unrelated.

:+1:

**Comment by yvaucher** [comment]
:+1: restarting travis

**Comment by yvaucher** [comment]
We are green

**Pull request merged**